### PR TITLE
wp-cli spec file for v1.1.0

### PR DESCRIPTION
**Pull request by SosyalAlkolik**
I added the spec file.

See #3793

### CODE REVIEW

File: utils/wp-cli-1.1.0.spec [hunk 1]
```diff
@@ -0,0 +1,45 @@
+%define Source1 /home/`whoami`/rpmbuild/SOURCES/wp.1.gz
```
Comment (danielbachhuber) [inline utils/wp-cli-1.1.0.spec]: Could we rename this file to `utils/wp-cli-rpm.spec`, so we don't need to rename it in each release?

File: utils/wp-cli-1.1.0.spec [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+Name:		wp-cli
+Version:	1.1.0
+Release:	1%{?dist}
+
+Summary:	The command line interface for Wordpress
```
Comment (danielbachhuber) [inline utils/wp-cli-1.1.0.spec]: This should be:

> The command line interface for WordPress

**Comment by SosyalAlkolik** [comment]
I changed the name of file.

**Comment by danielbachhuber** [comment]
Thanks @SosyalAlkolik !

**Comment by szepeviktor** [comment]
@SosyalAlkolik You may cherry-pick from https://github.com/wp-cli/wp-cli/pull/3800/files

**Comment by danielbachhuber** [comment]
@szepeviktor Could you clarify?

**Comment by szepeviktor** [comment]
I am not much of an RPM guy but the closed PR seems to have more details.

**Pull request merged**